src/main/resources/db/migration/V1__init.sql

```diff
@@ -1,6 +1,33 @@
 DROP TABLE IF EXISTS posts;
+DROP TABLE IF EXISTS users;
+DROP TABLE IF EXISTS comments;
+DROP TABLE IF EXISTS likes;
+
+
+CREATE TABLE users (
+  id bigserial PRIMARY KEY,
+  email varchar (80) NOT NULL,
+  password varchar (12) NOT NULL,
+  name varchar (20) NOT NULL
+);
 
 CREATE TABLE posts (
   id bigserial PRIMARY KEY,
-  content varchar(250) NOT NULL
+  content_post varchar(250) NOT NULL,
+  user_id  integer references users(id),
+  post_date_time timestamp
```

Comment:
Should we have a more consistent naming style/convention?
ie. should `content_post` be named `post_content` to match the `post_date_time`

Reply:
For simplicity/readability of the code, I think we'll change everything to just "content", then we can refer to it as post.content in the java file. @shwetzpatil  do you agree?